spec.bs

```diff
@@ -1680,13 +1682,17 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
           1. Let |directFromSellerSignalsForBuyer| be the result of running
             [=get direct from seller signals for a buyer=] with |directFromSellerSignals|, and |ig|'s
             [=interest group/owner=].
-          1. Let |dataVersion| be |trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/data versions=]
-             [|ig|'s [=interest group/name=]].
+          1. Let |dataVersion| be null.
+          1. Let |allTrustedBiddingSignals| be null.

```

Comment:
nit: just for clarity, maybe write `be an [=ordered map=]-or-null, initially null.`

Reply:
Done